### PR TITLE
Feature/np 2358 unified logging search

### PR DESCRIPTION
**Pull request by Carmendelope**
#### What does this PR do?
- Update the common functions in decorator to manage arrays instead of structs.
- Update sorting-decorator to include public_api_go.LogEntryResponse.
- Update search command to include new fields. 
- Comment all the code of the `monitor` command. @Thiamath has a branch open with this.
#### What are the relevant tickets?

- [NP-2358](https://nalej.atlassian.net/browse/NP-2358)

### CODE REVIEW

File: cmd/public-api-cli/commands/unified-logging.go [hunk 1]
```diff
@@ -39,9 +39,13 @@ func init() {
 	_ = logCmd.MarkFlagRequired("organizationID")
 
 	logCmd.AddCommand(searchCmd)
+	searchCmd.Flags().StringVar(&descriptorID, "descriptorID", "", "Application descriptor identifier")
 	searchCmd.Flags().StringVar(&instanceID, "instanceID", "", "Application instance identifier")
-	_ = searchCmd.MarkFlagRequired("instanceID")
+	//_ = searchCmd.MarkFlagRequired("instanceID")
```
Comment (dhiguero) [inline cmd/public-api-cli/commands/unified-logging.go]:
```suggestion
```

File: internal/pkg/server/decorators/common.go [hunk 2]
```diff
@@ -87,11 +91,37 @@ func FromAppDescriptorList(result []*grpc_application_go.AppDescriptor, decorato
...
+	}
+
+	// reconvert to grpc_public_api_go.LogEntryResponse
+	orderedResult := make([]*grpc_public_api_go.LogEntryResponse, len(result))
+	for i, d := range ordered {
+		aux := d.(grpc_public_api_go.LogEntryResponse)
+		orderedResult[i] = &aux
+	}
+
+	log.Debug().Interface("orderedResult", orderedResult).Msg("FromLogEntryResponse")
```
Comment (dhiguero) [inline internal/pkg/server/decorators/common.go]:
```suggestion
```

File: cmd/public-api-cli/commands/unified-logging.go (hunk 1 above)
Comment (Carmendelope) [reply cmd/public-api-cli/commands/unified-logging.go]: Removed

File: internal/pkg/server/decorators/common.go (hunk 2 above)
Comment (Carmendelope) [reply internal/pkg/server/decorators/common.go]: Removed

**Pull request merged**